pkg/handler/handler.go

```diff
@@ -323,11 +337,6 @@ func (m *Modifier) Handle(w http.ResponseWriter, r *http.Request) {
 			body = data
 		}
 	}
-	if len(body) == 0 {
```

Comment:
Any reason this was removed?

Reply:
Yea, this error wasn't helpful and didn't return an `AdmissionReview` I could add it back and just return an actual `AdmissionReview` with a more helpful error message instead of attempting to process the empty body